test/firmata.test.js

```diff
@@ -515,6 +515,51 @@ describe("board", function() {
     serialPort.emit("data", [ANALOG_MESSAGE | (1 & 0xF), 0 % 128, 0 >> 7]);
   });
 
+  it("should be able to read value of analog pin on a board that skipped capabilities check", function(done) {
+    var serialPort = new SerialPort("/path/to/fake/usb");
+    var board = new Board(serialPort, {skipCapabilities: true, analogPins: [14,15,16,17,18,19]}, function(err) {});
```

Comment:
add another test for reading an analog pin when skipCapabilities is true but no analogPin map is passed in. It should still work. If the test fails we may be able to make it pass buy making some changes to firmata.js since the analog pin map shouldn't affect analog reads unless I'm overlooking something.